docs/index.md

```diff
@@ -5,19 +5,55 @@ _navPath: toc.html
 _navRel: ../toc.html
 ---
 
-## Getting started
+.NET Documentation
+==================
 
-In this docset, you will find tutorials that will teach you how to get started with .NET Core. You can also find the API reference documentation. 
+Welcome to .NET! You can build any kind of app you want with .NET, from cloud to mobile to desktop to IoT to games. You can start building your next app today, for free, on Windows, OS X and Linux.
```

Comment:
open source and free (as in beer) don't have much to do with one another. Swift is open source but you need to pay $99 to submit an app. There are lots of examples like that. The thing that is open source is open source and free (as in beer) but it may not be the thing you think it is.